pkg/apis/eventsource/v1alpha1/context.go

```diff
@@ -0,0 +1,18 @@
+package v1alpha1
+
+// Context holds a general purpose REST API context
+type Context struct {
```

Comment:
`Context` might be too generic, was it specifically for `webhook`? maybe `WebhookContext`?  @VaibhavPage

Reply:
it was only used here - I could move to common?

Reply:
I think moving it here is ok, but giving a name `WebhookContext` might be better because it is only used for webhook related things.

Reply:
done